tests/key_field/key_field_sample_generated.h

```diff
@@ -80,33 +70,16 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(1) Baz FLATBUFFERS_FINAL_CLASS {
   uint8_t b() const {
     return flatbuffers::EndianScalar(b_);
   }
-  void mutate_b(uint8_t _b) {
-    flatbuffers::WriteScalar(&b_, _b);
-  }
 };
 FLATBUFFERS_STRUCT_END(Baz, 5);
 
-inline bool operator==(const Baz &lhs, const Baz &rhs) {
```

Comment:
May I know why operators got removed ?

Reply:
Sorry I didn't mean to delete them, they are removed because when I run the generator, I didn't run with all the flags required for these, I'll regenerate them.